hooks/post-command

```diff
@@ -10,15 +10,28 @@ if [[ ${BUILDKITE_COMMAND_EXIT_STATUS:-0} != '0' ]]; then
   exit 0
 fi
 
+if ! aws sts get-caller-identity; then
+  echo "Unable to invalidate cloudfront cache: credentials missing"
+  exit 1
+fi
```

Comment:
I'm not sure this adds any value, we will almost always have credentials, but those credentials may or may not have permission to call `create-invalidation`